core/src/network.ts

```diff
@@ -31,6 +31,22 @@ const SUPPORTED_NETWORKS: NetworkConfig[] = [
     },
 ];
 
+export const getDevelopmentNetworkConig = function (alsoSetState = false): NetworkConfig {
```

Comment:
Since you've used `setupRpcUrlAndGetNetworks` and now only refer to `custom` network, I understand that this function can be deleted, right?